course-contents.md

```diff
@@ -54,21 +55,18 @@
 5.	Participants can produce charts to display statistics, including bar, line, scatter
 
 *Content:*
-
-3.1. Exploring data structures **(Stu)**
-    -	Same stuff they cover in the dplyr section of the R course:
-    - Selecting columns
-    - Filtering
-    - Grouping/summarizing -> .agg
-    - Counts -> value_counts
-    - Mutating (assign)
-    - ‘Piping’ into a single go to avoid temporary dataframes
-    - Joins 
-
-3.2. Summary stats / Statistical tests **(Lindsay)**
-
-3.3. Graphical depictions of data **(Stu)**
-   -	e.g. Matplotlib and/or Seaborn 
+3.1. Exploring data structures with pandas **(Stu)**
+    - Selecting rows and columns
+    - Grouping rows and columns
+    - Sorting data
+    - Creating columns (transform)
+    - Joining datasets
+    - Descriptive stats methods (e.g. info(), describe(), value_counts, sum(), agg(), etc.)
```

Comment:
Are you planning to include mutates/assigning and 'piping?' (I'd probably call it chaining in python). I think those are the only things off the old list you don't have on your new pythonic list.

Reply:
Yes, i will include that in 3.1! thanks